requirements/base.txt

```diff
@@ -0,0 +1,7 @@
+pip==8.1.2
+bumpversion==0.5.3
+wheel==0.29.0
+watchdog==0.8.3
+django==1.9.7
```

Comment:
Rather than pinning, we probably want to do a >= here, to avoid incompatibilities with the dependencies specified in the app (e.g. Kolibri).